glossary.po

```diff
@@ -737,24 +736,20 @@ msgstr ""
 "`__eq__`. En Perl, les dictionnaires sont appelés \"*hash*\"."
 
 #: glossary.rst:311
-#, fuzzy
 msgid "dictionary comprehension"
-msgstr "liste en compréhension (ou liste en intention)"
+msgstr "liste en compréhension (ou liste en intention) d'un dictionnaire"
```

Comment:
dictionnaire en compréhension ?

Reply:
Je trouve ca pas beau :/

Reply:
OK, mais « liste en compréhension (ou liste en intention) d'un dictionnaire » ce n'est pas ça.
On parle de constructions comme :
`{n: n ** 2 for n in range(10)}`
Il n'y a pas d'idée de liste ici.